srcpkgs/python-PyAudio/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'python-PyAudio'
+pkgname=python-PyAudio
+version=0.2.11
+revision=1
+noarch=yes
+wrksrc="PyAudio-${version}"
+build_style=python-module
+pycompile_module="PyAudio.py"
+hostmakedepends="python-setuptools python3-setuptools"
+makedepends="python-devel python3-devel portaudio-devel"
+depends="python"
```

Comment:
Remove `depends`. After removing `noarch=yes`, dependencies will be auto-detected by xbps-src:
```
   SONAME: libportaudio.so.2 <-> portaudio>=19.20140130_1
   SONAME: libpython3.6m.so.1.0 <-> python3>=3.6.2_1
```